buildSrc/src/main/kotlin/Checkstyle.kt

```diff
@@ -29,6 +29,10 @@ import org.gradle.kotlin.dsl.withType
 class NessieCheckstylePlugin : Plugin<Project> {
   override fun apply(project: Project): Unit =
     project.run {
+      if (project.name.endsWith("-proto")) {
+        return
```

Comment:
Why not run checkstyle on proto modules?

Reply:
it's generated code

Reply:
(Try it, if your terminal's scroll buffer is huuuuuge ;) ;) )

Reply:
> it's generated code

Could you add a comment about that?

What about `.proto` files themselves?

Reply:
Checkstyle is Java only